xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -111,8 +111,8 @@
 #define ADDON_INSTANCE_VERSION_SCREENSAVER_XML_ID     "kodi.binary.instance.screensaver"
 #define ADDON_INSTANCE_VERSION_SCREENSAVER_DEPENDS    "addon-instance/Screensaver.h"
 
-#define ADDON_INSTANCE_VERSION_VFS                    "2.1.0"
-#define ADDON_INSTANCE_VERSION_VFS_MIN                "2.1.0"
+#define ADDON_INSTANCE_VERSION_VFS                    "2.2.0"
+#define ADDON_INSTANCE_VERSION_VFS_MIN                "2.2.0"
```

Comment:
Why are we bumping MIN version for this ABI compatible change?

Reply:
@ksooo : My bad, happened during rebase. Will fix it asap.

Reply:
@ksooo : Fixed